### ISSUE TITLE
Initial tilt doesn't work (tilts to a random XY on refresh)

### ISSUE THREAD

**Issue by shunkakinoki** [mkosir/react-parallax-tilt#8]
### 🐛 Issue/Bug Report Template

**Describe the bug**  

Thanks for this amazing package!

**To Reproduce**  

1. Go down to the <Tilt>
2. refresh site
3. The tilt doesn't comply with `reset` or `tiltAngleXInitial`, `tiltAngleYInitial`

**Expected behavior**  

**Link to demo (highly encouraged)**  

Reproducible example coming soon!

**Code snippets**  

**Environment**

Chrome

**Additional context**

**Comment by rardoz** [comment]
Workaround - including an onEnter prop into the tilt component fixes it. 

My code:
```
<Tilt onEnter={element => { console.log(element) }} gyroscope scale={1.3}>
```

**Comment by mkosir** [comment]
Hi @shunkakinoki, @rardoz thanks for the feedback and suggested fix, I'm glad you like the package!
Bug fix is merged and deployed in the newest version `1.3.43`, please check and confirm if it's working for you as expected?

**Comment by shunkakinoki** [comment]
@mkosir Just updated and checked this right now!
I can confirm the bug is completely fixed - works perfectly now!!! 
Thank you, I really appreciate your work!!!
It's honestly the best library I've encountered this year, and wish you the best
Thanks again for your dedication. Respect.
